pyproject.toml

```diff
@@ -13,7 +13,7 @@ license = {text = "Apache 2"}
 dependencies = [
     "fms-model-optimizer>=0.2.0",
     "ibm-fms==1.0.0",
-    "vllm>=0.9.0",
+    "vllm>0.9.0,!=0.9.1",
```

Comment:
Do we need to say `>0.9.0`? I thought we still worked with `0.9.0`

Reply:
Ah, nice catch!
Fixed